climpred/bootstrap.py

```diff
@@ -51,6 +51,44 @@ def _resample(hind, resample_dim):
     return smp_hind
 
 
+def _resample_idx(init, bootstrap, dim='member', replace=True):
```

Comment:
1. I think this should be renamed to something more descriptive. I can't think of anything great right now. But the function serves to resample and append the resampled ensemble to a new dimension (as opposed to returning a single resampled instance). Maybe `_resample_in_bulk` or `_return_resampled_ensemble`. Any ideas?

Reply:
2. I think `bootstrap` should be called `iterations`. This whole process is bootstrapping. I think it's more clear to have the argument be the number of iterations being chosen.

Reply:
`_resample_iterations_idx` and changed to `iterations` and dim `iteration`